lua/popup/init.lua

```diff
@@ -94,7 +94,18 @@ function popup.create(what, vim_options)
 
   if vim_options.line then
     -- TODO: Need to handle "cursor", "cursor+1", ...
-    win_opts.row = vim_options.line
+    if type(vim_options.line) == 'string' then
+      assert(string.find(vim_options.line,'^cursor') ~= nil, "Invalid option")
+      local line = vim.fn.getcurpos()[2]
```

Comment:
In general, I think we should try and use `vim.api` for these things.

Would something like `vim.api.nvim_win_get_cursor(0)` work?